### PR TITLE
Add Question: Organization and Scrum

### PR DESCRIPTION
**Pull request by huserben**
Reference Issue: #50

### CODE REVIEW

File: questions/organization_and_scrum.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: "Organization and Scrum"
+parent: Katas
+---
+The organization's management asks you what they should do for the Scrum Team. As a Scrum Master, what should you answer?
```
Comment (henadzkavaliou) [inline questions/organization_and_scrum.md:6]:
```suggestion
The organization's management asks you what they should do for the Scrum Team. As a Scrum Master, what might you answer?
```

Comment (huserben) [reply questions/organization_and_scrum.md:6]: I like the first version as I believe it provides more guidance and makes a good question (and I'd be very curious to hear the different answers to that 😄 )

Comment (henadzkavaliou) [reply questions/organization_and_scrum.md:6]: I'll reach out to Manolo on Monday and make sure he doesn't mind adjusting his question.

File: questions/organization_and_scrum.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: "Organization and Scrum"
```
Comment (henadzkavaliou) [inline questions/organization_and_scrum.md:3]:
```suggestion
title: "Guidance to top management"
```

**Review by henadzkavaliou — APPROVED** [review]

**Pull request merged**